py3status/modules/group.py

```diff
@@ -78,6 +78,9 @@
 ```
 
 @author tobes
+
+SAMPLE OUTPUT
+{'full_text': 'output of another module'}
```

Comment:
I think `module 1` would look clean for screenshot... similar to frame's `module 1|module 2|module 3`